website/docs/r/tenant.html.markdown

```diff
@@ -13,6 +13,7 @@ Manages MSO Tenant
 ## Example Usage ##
 
 ```hcl
+
 resource "mso_tenant" "tenant1" {
```

Comment:
The rest of this resource id's should be also changed to point to right resource ids

Reply:
ok